styles/sections/_mission.scss

```diff
@@ -0,0 +1,106 @@
+@use '../reset';
+@use '../colors';
+
+// TO-DO: ADD IN BACKGROUND IMAGES
+/* #mission {
+  position: relative;
+  height: 100%;
+} */
+
+.mission-header-goals-wrapper {
+  // TO-DO: ADD IN BACKGROUND IMAGES
+  /* &:before {
+    content: ' ';
+    display: block;
+    position: absolute;
+    top: 545px;
+    left: 0;
+    width: 200px;
+    height: 200px;
+    background: url('./assets/stairs-wireframe.svg') no-repeat;
+  } */
+
+  p {
+    @container body (min-width: 1024px) {
+      max-width: 600px;
+      margin-left: 100px;
+    }
+  }
+
+  ul {
+    max-width: 380px;
+    margin: 0 auto;
+
+    @container body (min-width: 1024px) {
+      max-width: unset;
+      display: inline-grid;
+      grid-template-columns: 1fr 1fr 1fr;
+      column-gap: 50px;
+      place-items: start;
+      position: relative;
+    }
+  }
+
+  li {
+    p {
+      @container body (min-width: 1024px) {
+        margin-left: 0;
+      }
+    }
+  }
+}
+
```

Comment:
Is this style to undo the `margin-left: 100px;` style for paragraph tags on line 23? if so, I wonder if we can clean this up and make the css starting on line 23 more specific to only target the paragraph text in the mission header?